utils/AnalysisUtils.py

```diff
@@ -642,3 +643,63 @@ def ComputeRatioDiffBins(hNum, hDen, uncOpt=''):
     hRatio.Divide(hNumReb, hDenReb, 1., 1., uncOpt)
 
     return hRatio
+
+
+def ScaleGraph(graph, scaleFactor):
+    '''
+    Helper method to scale a TGraph
+
+    Parameters
+    ----------
+
+    - graph: graph to scale
+    - scaleFactor: scale factor
+    '''
+    for iPt in range(graph.GetN()):
+        x, y = Double(), Double()
+        graph.GetPoint(iPt, x, y)
```

Comment:
A minor comment. I discovered today running `ComputeEfficiencyDplusDs.py` that `Double()` is deprecated since ROOT v6.20. The suggested way to pass a variable by reference to PyROOT is:
```python
import ctypes
x_ref, y_ref = ctypes.c_double(), ctypes.c_double()
graph.GetPoint(iPt, x_ref, y_ref)
x = x_ref.value
y = y_ref.value
graph.SetPoint(iPt, x, y * scaleFactor)
```
In any case `Double()` still works and maybe it is necessary to check if this new method works also for older ROOT versions.

Reply:
I also noticed that, and in the first instance I used it, but I got the following error:
```python3
  File "../utils/AnalysisUtils.py", line 701, in DivideGraphByHisto
    ratioUncLow = numUncLow/num * num/den
TypeError: unsupported operand type(s) for /: 'float' and 'c_double'
```
So, I did not investigated further, and I sticked to the ```Double()``` for the moment, but if you prefer we can try to fix it and use the ```c_double```

Reply:
Additional info, if I define everything as ```c_double()```, I get a similar error:
```python3
  File "../utils/AnalysisUtils.py", line 701, in DivideGraphByHisto
    ratioUncLow = numUncLow/num * num/den
TypeError: unsupported operand type(s) for /: 'c_double' and 'c_double'
```

Reply:
I think that you should do:
```python
x = c_double()
# x passed by reference to a function
b = x.value * 2
```
so `c_double()` is "similar" to a pointer and you should "dereference" it with `.value` to have the number.

Reply:
Thanks!